### ISSUE TITLE
How to run it after the minifier?

### ISSUE THREAD

**Issue by customautosys** [elmeet/vite-plugin-javascript-obfuscator#13]
I'm running vite-plugin-javascript-obfuscator as a plugin in my Quasar vite project. However, the problem is that after it runs, the minifier runs which results in the file being un-obfuscated as the variable names are mangled back (i.e. not 0xa... but a, b, c etc.)

How do I make the obfuscator run after the minifier so that the obfuscator's effects remain?

**Comment by elmeet** [comment]
There are some conflicts between minifier and `javascript-obfuscator`.

Unfortunately, based on the [vite plugin ordering](https://vitejs.dev/guide/api-plugin.html#plugin-ordering), currently the plugin cannot run after the minifier.

**Comment by customautosys** [comment]
> There are some conflicts between minifier and `javascript-obfuscator`.
> 
> Unfortunately, based on the [vite plugin ordering](https://vitejs.dev/guide/api-plugin.html#plugin-ordering), currently the plugin cannot run after the minifier.
> 
> 

Hm... OK so how do I make the final output obfuscated and also with all the whitespace minified?

**Comment by elmeet** [comment]
You can try to modify this configuration, but this will affect all minified results.

vite.config.js
```javascript
export default defineConfig({
  build: {
    minify: "terser",
    terserOptions: {
      mangle: false,
    },
  },
})
```
or
```javascript
export default defineConfig({
  build: {
    minify: "esbuild",
  },
  esbuild: {
    minifyIdentifiers: false,
  },
})
```

**Comment by customautosys** [comment]
I tried it and managed to get the terser working with
```typescript
{
	mangle:false,
	keep_classnames:true,
	keep_fnames:true
}
```
Any idea how anybody can get stringArray working? Until now stringArray breaks my code till it does not work at all 100% of the time.

Or is there any way to encode the strings in base64 or rc4 without using stringArray?

Actually, I also found this option, enforce:'post'. Is there a way to build it into the plugin itself (https://vitejs.dev/guide/using-plugins.html#enforcing-plugin-ordering)? (oh wait, that won't work because it'll still run before the minifier)
